### PR TITLE
Simplify installation instructions

### PR DESCRIPTION
**Pull request by FND**
Merely a suggestion, but I figured this might scan better.

PS: I was made aware of Globalize by @alolita's [presentation](https://www.youtube.com/watch?v=N0BeFY3B2sI). 🙇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,18 +145,21 @@ Read the [Locales section](#locales) for more information about supported locale
...
+#### Downloading a ZIP or tarball archive
 
-Click the github [releases tab](https://github.com/globalizejs/globalize/releases) and download the latest available Globalize package.
+Click the GitHub [releases tab](https://github.com/globalizejs/globalize/releases) and download the latest available Globalize package.
 
-*By using a package manager...*
+#### Using a package manager
 
-Use bower `bower install globalize`, or npm `npm install globalize`.
+You can use either npm and Bower:
```
Comment (rxaviers) [inline README.md]: One minor grammar correction suggestion: [either ... or](https://dictionary.cambridge.org/us/grammar/british-grammar/linking-words-and-expressions/either-or):

```suggestion
You can use either npm or bower:
```

**Comment by rxaviers** [comment]
Thanks for your PR and for sharing @alolita's presentation. I just added a small ask so that I can merge it. Also, please sign the CLA (instructions above)

File: README.md (hunk 1 above)
Comment (FND) [reply README.md]: Oops, good catch - fixed retroactively in 5b816a90eb754fd92af2ac167c0960edaebf0152.

CLA should be signed as well now.

**Comment by rxaviers** [comment]
Thanks!

**Pull request merged**